featuretools/tests/primitive_tests/test_datetoholiday_primitive.py

```diff
@@ -109,3 +109,26 @@ def test_multiple_countries():
     ]
     for x in countries:
         DateToHoliday(country=x)
+
+
+def test_with_timezone_aware_datetimes():
+    df = pd.DataFrame(
+        {
+            "non_timezone_aware_with_time": pd.date_range(
+                "2018-07-03 09:00", periods=3
+            ),
+            "non_timezone_aware_no_time": pd.date_range("2018-07-03", periods=3),
+            "timezone_aware_with_time": pd.date_range(
+                "2018-07-03 09:00", periods=3
```

Comment:
should we test with times where if converted to UTC, some times would change dates?